prow/plugins/shrug/shrug.go

```diff
@@ -47,7 +48,18 @@ type event struct {
 }
 
 func init() {
-	plugins.RegisterGenericCommentHandler(pluginName, handleGenericComment, nil)
+	plugins.RegisterGenericCommentHandler(pluginName, handleGenericComment, helpProvider)
+}
+
+func helpProvider(config *plugins.Configuration, enabledRepos []string) (*pluginhelp.PluginHelp, error) {
+	// The Config field is omitted because this plugin is not configurable.
+	return &pluginhelp.PluginHelp{
+			Description: "The shrug plugin applies and removes the shruggie label.",
+			WhoCanUse:   "Anyone can use the shrug plugin commands as long as they physically shrug while doing so.",
```

Comment:
> as long as they physically shrug while doing so.

I thought we were going to keep this undocumented but I like this requirement.

Reply:
I'm happy to keep it undocumented if that is preferable, but it will appear on the plugin-help page even if it doesn't specify a help providing func (it will show that it is enabled in the appropriate repos, but that help could not be loaded for the plugin).
So the plugin help page might blow its cover either way.
/shrug

Reply:
We could blacklist this plugin with a secret plugins option 🤔

Reply:
I agree with the physical requirement. Very important. But +1 on keeping undocumented.

Even just replacing what it does with `<redacted>` or something

Reply:
@rmmh suggested that the best way to hide it is with a css rule since no one looks at css and it is easy to add. The `shrug` plugin doesn't appear at all now.